iamproperty/iamproperty.github.io
# Ds 61/news promo carousel

## Summary of Changes:
1. Create carousel JS
2. New CSS
3. Unit tests

## Rationale
Create the ability to have a carousel which can show either one slide or multiple at a time.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/carousel, /examples/news-promo-cards and /examples/logo-carousel

## Ticket(s)
DS-61
DS-57

## Checklist

## Notes
I have used scroll-snap features instead of the default trying to re-purpose bootstraps carousel. This allows us to use the grid system and allows for multiple slide in view at once.
